lib/fathead/mdn_css/parse.pl

```diff
@@ -380,7 +380,7 @@ sub _category_title {
 sub _clean_string {
     my $input = shift;
     say "Input: '$input'";
-    $input =~ s/[:<>@()]//g;
+    $input =~ s/[:<>@()]/ /g;
```

Comment:
Though this works, I'm not sure the approach is correct here.

This line is meant to strip away these characters -- it's not clear why we'd want to replace them all with spaces instead.

It may be better to replace `(:?:?-moz.+)\((.+)\)` with `$1 ($2)` first, then let this line strip away those characters